src/app/domain/geolocation/hooks/tracking.ts

```diff
@@ -45,4 +59,19 @@ export const useGeolocationTracking = (): void => {
 
     void initializeTracking()
   }, [client])
+
+  const webhookQuery = buildServiceWebhookQuery()
+  const webhookResp = useQuery(webhookQuery.definition, webhookQuery.options)
+
+  if (Array.isArray(webhookResp.data) && webhookResp.data.length > 0) {
```

Comment:
nit: You can webhookResp?.data?.length > 0

Reply:
I actually added the Array.isArray because typescript was complaining with: `Property 'length' does not exist on type 'object | any[]'.`